guides/common/modules/proc_configuring-pull-based-transport-for-remote-execution.adoc

```diff
@@ -0,0 +1,17 @@
+[id="configuring-pull-based-transport-for-remote-execution_{context}"]
+= Configuring Pull-based Transport for Remote Execution
+
+Use this procedure to set up `pull-mqtt` mode for pull-based remote execution transport.
+
+.Procedure
+ifdef::satellite,orcharhino,foreman[]
```

Comment:
```suggestion
ifdef::satellite,orcharhino,katello[]
```

Reply:
This is essentially manual configuration, which should only be needed in non-katello scenarios, so only `foreman`.

Reply:
@adamruzicka To confirm, are you saying that no changes need to be made or that I need to remove satellite and orcharhino?

Reply:
Remove satellite and orcharhino